alpha/lib/model/UserEntryPeer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+/**
+ * Skeleton subclass for performing query and update operations on the 'user_entry' table.
+ *
+ * Describes the relationship between a specific user and a specific entry
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package Core
+ * @subpackage model
+ */
+class UserEntryPeer extends BaseUserEntryPeer {
+
+	const QUIZ_OM_CLASS = 'QuizUserEntry';
```

Comment:
Please remove this constant, the word "quiz" shouldn't appear anywhere outside the Quiz plugin.